spec.bs

```diff
@@ -579,46 +580,77 @@ An auction config is a [=struct=] with the following items:
 <dl dfn-for="auction config">
 : <dfn>seller</dfn>
 :: An [=url/origin=].
+  The origin of the seller running the ad auction.
 : <dfn>decision logic url</dfn>
 :: A [=URL=].
+  The URL to fetch the seller's JavaScript from.
 : <dfn>trusted scoring signals url</dfn>
 :: Null or a [=URL=].
+  Provide a mechanism for making real-time data available for use at scoring time.

```

Comment:
Can you say anything about the expected content of the URL? Or if the content is passed verbatim to the scoring script, you could say that.